test/fail_compilation/README.md

```diff
@@ -0,0 +1,21 @@
+# Tests for diagnostics on compilation failure
+
+Each D file will be compiled and the compilation is expected to fail.
```

Comment:
I propose to extend the sentence with a small explanation on the return codes.

"... with a return code of 1. Most of the times return code of 0 indicates that a diagnostic is broken (new case of "accept invalid") and over 1 that there is a new case ICE (e.g 139)."

Reply:
allright, let's say the mini change is ok. I'll add more details latter if I still feel it.

Reply:
Added the expected exit code but the explanation for sucessfull compilation seems redundant.

Also, ICE 's usually don't exit > 1 (assertion failures are 1 and other failures like segfaults have their platform defined defaults).